gradle.properties

```diff
@@ -56,8 +56,8 @@ ZCASH_VERSION_NAME=0.1
 # available on Google Play. This is useful for testing, or for a forked version of the app.
 ZCASH_RELEASE_APP_NAME=sECCant
 ZCASH_RELEASE_PACKAGE_NAME=co.electriccoin.zcash
-
 ZCASH_DEBUG_APP_NAME_SUFFIX=" (D)"
+ZCASH_SUPPORT_EMAIL_ADDRESS=
```

Comment:
Don't we want to have this property initialized with our value by default? This way we need to keep it somewhere. And I see this property the same as those above (e.g. package or app names). A developer, who forks this repository will need to change those too. Or you see it a bit more important, so we have here a more save variant - no default value?

Reply:
It was intentional.  I think the email is more dangerous, in the sense that we don't want to be getting flooded with support emails for apps we didn't create.  A fresh checkout of the app will function without this being present, as the in-app support screen is a minor feature overall.

This one is different from package and app name.  Others cannot release an app with our package name, because the package name is a global namespace on Google Play. So others will be forced to change it.  App name is also pretty obvious—people will want to change it right away since it is visible on the home screen and people will want their own brand if they fork the app.

Any other alternatives you can think of?  I preferred not using a fake email, because that seemed worse for user privacy/security.

Reply:
Yep, understand. I'm just thinking about having it somewhere in the project and don't have to (and also don't forget to) set it up while releasing the app. Could CI script do it for us (e.g. as part of secrets)?

Reply:
This is done as part of the CI job, and is included in the changes for this PR.  Is there something else specifically that's missing?

Reply:
Yes, I see. I just added a little documentation comment about the new vars field. Strip it out, if it's not needed.